pkg/config/helpers.go

```diff
@@ -0,0 +1,17 @@
+package config
+
+import (
+	"fmt"
+	"os"
+
+	"gopkg.in/check.v1"
+)
+
+func GetEnvOrSkip(c *check.C, varName string) string {
```

Comment:
Is this method already defined in testutil?

Reply:
https://github.com/kanisterio/kanister/blob/master/pkg/testutil/fixture.go#L70

Reply:
@tdmanv Yes, It creates a cycle if I import `pkg/testutil`, hence I added the function to `helpers.go` 
Also, this func is defined in 4 different places in kanister 😄 . Once this PR is merged, I can push another followup PR to remove the duplicate funcs and import only from `helpers.go`.

Reply:
that works for now